### DISCUSSION TITLE
Can't upgrade from 11.0.11 to 11.1.3

### DISCUSSION THREAD

**Discussion by tkefauver** [AvaloniaUI/Avalonia#17081] [Q&A] [answered]
I've been away from Avalonia for a while and tried running 11.1.3 (on Windows) but am getting lots of problems from style and resource loading.

Simplest example is setting a button theme inline like this results in an empty style (no bg or hover behavior):
```
<Button HorizontalAlignment="Left">
    <Button.Resources>
        <SolidColorBrush x:Key="ButtonBackground">Gold</SolidColorBrush>
        <SolidColorBrush x:Key="ButtonBackgroundPointerOver">Yellow</SolidColorBrush>
        <SolidColorBrush x:Key="ButtonBackgroundPressed">White</SolidColorBrush>

        <SolidColorBrush x:Key="ButtonForeground">Black</SolidColorBrush>
        <SolidColorBrush x:Key="ButtonForegroundPointerOver">Black</SolidColorBrush>
        <SolidColorBrush x:Key="ButtonForegroundPressed">Black</SolidColorBrush>
    </Button.Resources>
    <TextBlock Text="Button Text" />
</Button>
```

Also I get weird behavior all over the place. Animations, images and colors not loading (but some do?) but no binding errors besides a string to FontFamily conversion exception which makes me think something has changed with global style loading. 

Everything works fine in 11.0.11 (went directly to 11.1.3).  Has there been changes to how styles are being loaded?

**Comment by rabbitism** [comment]
Any third party controls?

**Reply by tkefauver** [reply]
A couple, [Avalonia.WebView](https://github.com/MicroSugarDeveloperOrg/Avalonia.WebView) and [Avalonia.HtmlRenderer](https://github.com/AvaloniaUI/Avalonia.HtmlRenderer) but the issues come up before those even load.

**Reply by tkefauver** [reply, marked as answer]
Seems this exception was the root cause:
`Unable to cast object of type 'System.String' to type 'Avalonia.Media.FontFamily`

Get them anytime a window is closed which never happened before (<11.1.2) but doing this in a `Closing` handler prevents it:
`window.FontFamily =  FontFamily.Default;`

Not sure what's going on under the hood but maybe a custom font bug during window disposal not sure though.
